scripts/common.py

```diff
@@ -53,7 +53,11 @@
 
 # Versions of libraries used in backend.
 PILLOW_VERSION = '6.2.2'
-REDIS_CLI_VERSION = '6.0.6'
+# Redis 6.0.5 not latest stable build of redis (6.0.6) because there is a
+# 'make test' bug where the solution has not been released explained in this
+# issue: https://github.com/redis/redis/issues/7540.
+# TODO: Upgrade to stable build when the make test bug is resolved.
```

Comment:
TODOs always come with issue numbers, please follow existing conventions.

That said: you probably don't need this TODO at all tbh, assuming 6.0.5 is no less stable than 6.0.6.

Reply:
I see fixed.

Reply:
Oh I haven't actually pushed the changes for this yet. Since I'm trying to figure out some other issues. I will push it asap.